src/Agent.Plugins/GitSourceProvider.cs

```diff
@@ -347,7 +356,6 @@ public abstract class GitSourceProvider : ISourceProvider
             if (selfManageGitCreds)
```

Comment:
is this now an empty if statement?

Reply:
Good catch, I have no idea what happened there. Unintentional delete!